src/classes/CRLP_RollupBatch_SVC.cls

```diff
@@ -125,10 +127,9 @@ public class CRLP_RollupBatch_SVC {
      * @return True or False
      */
     public static Boolean doesRecordNeedSkewRollupMethod(Id recordId, CRLP_RollupProcessingOptions.RollupType jobType) {
-        if (jobType == CRLP_RollupProcessingOptions.RollupType.AccountHardCredit ||
-                jobType == CRLP_RollupProcessingOptions.RollupType.ContactHardCredit ||
-                jobType == CRLP_RollupProcessingOptions.RollupType.AccountContactSoftCredit ||
-                jobType == CRLP_RollupProcessingOptions.RollupType.ContactSoftCredit) {
+        if (jobType != CRLP_RollupProcessingOptions.RollupType.GAU &&
```

Comment:
Is there any fear of a RollupType being added that we wouldn't want included here?

Reply:
@lparrott Rollup types are not only fairly stable, but also require substantial infrastructure to implement.  Any newly implemented rollup type will need integration in multiple locations and this is one of them.  I would expect anyone working on a new rollup type to understand the implications of how it should be implemented.  Flagging how records will be selected for inclusion is fundamental.

Reply:
@lparrott To your point, though, I intentionally added AccountSoftCredit rollup types to this selection.  I saw no good reason for their exclusion, but would like my thinking confirmed by @force2b.

Reply:
So, this went from explicit list to just not RD or GAU, meaning that the code in the if block runs for all other rollup types? That makes sense since the new `CustomizableRollups_UseSkewMode__c` field only exists on the Account and Contact objects - and all the other rollup types are for those two objects.